src/containers/App.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import { withRouter } from 'react-router-dom'
+import DragNDrop from "../components/DragNDrop";
+import DesignSearch from "../components/DesignSearch";
+import RunButton from "../components/RunButton";
+import TestDataArea from "../components/TestDataArea";
+import ErrorsArea from "../components/ErrorsArea";
+
+
+class App extends Component {
+
+    render() {
+        return (
+            <div className="container">
+                <div className="row">
```

Comment:
looks like you should use bootstrap-react components

Reply:
I was thinking about choosing between `bootstrap-react` and `bootstrap`